toolchain/check/testdata/let/no_prelude/import.carbon

```diff
@@ -8,65 +8,77 @@
 // TIP: To dump output, run:
 // TIP:   bazel run //toolchain/testing:file_test -- --dump_output --file_tests=toolchain/check/testdata/let/no_prelude/import.carbon
 
-// --- implicit.carbon
+// --- fail_implicit.carbon
 
 package Implicit;
 
+// CHECK:STDERR: fail_implicit.carbon:[[@LINE+4]]:5: error: `let` compile time bindings require function or interface (may be supported later)
+// CHECK:STDERR: let a:! () = ();
+// CHECK:STDERR:     ^~~~~~
+// CHECK:STDERR:
 let a:! () = ();
 
-// --- implicit.impl.carbon
+// --- fail_implicit.impl.carbon
 
 impl package Implicit;
 
+// CHECK:STDERR: fail_implicit.impl.carbon:[[@LINE+4]]:5: error: `let` compile time bindings require function or interface (may be supported later)
+// CHECK:STDERR: let b:! () = a;
+// CHECK:STDERR:     ^~~~~~
+// CHECK:STDERR:
 let b:! () = a;
 
-// --- other.carbon
+// --- fail_other.carbon
 
 package Other;
 
+// CHECK:STDERR: fail_other.carbon:[[@LINE+4]]:5: error: `let` compile time bindings require function or interface (may be supported later)
+// CHECK:STDERR: let a:! () = ();
+// CHECK:STDERR:     ^~~~~~
+// CHECK:STDERR:
 let a:! () = ();
 
-// --- import_other.carbon
+// --- fail_import_other.carbon
 
 import Other;
 
+// CHECK:STDERR: fail_import_other.carbon:[[@LINE+3]]:5: error: `let` compile time bindings require function or interface (may be supported later)
+// CHECK:STDERR: let b:! () = Other.a;
+// CHECK:STDERR:     ^~~~~~
 let b:! () = Other.a;
```

Comment:
Can we change these from `:!` to `:` bindings? It doesn't look like this test was trying to test compile time bindings in particular.

Reply:
Done